operator/controllers/syncer/csi_node.go

```diff
@@ -111,16 +115,15 @@ func GetCSIDaemonsetSyncer(c client.Client, scheme *runtime.Scheme, driver *csis
 			Value: envVars[k],
 		})
 	}
-
-	daemonMaxUnavailableGlobal = daemonSetMaxUnavailable
+	logger.Info("upgrade_straegty shouldn't be present in cmEnvVars", "cmEnvVars:", cmEnvVars)
```

Comment:
This log statement seems unnecessary as cmEnvVars will never have upgrade strategy due to condition at line no 105.